app/controllers/concerns/analytics.rb

```diff
@@ -2,23 +2,9 @@
 
 module Analytics
   def get_ui_analytics
-    ui_analytics_api_url = ENV['UI_URL'] + '/api/analytics'
+    ui_analytics_api_url = ENV['UI_URL'] + "/api/analytics?start_date=#{@start_date.to_s}&end_date=#{@end_date}"
```

Comment:
The `to_s` is redundant and also isn't used for the end_date handling, so it would be good to be consistent without the `to_s`

Reply:
done